pages/index.tsx

```diff
@@ -1,12 +1,22 @@
 import type { NextPage } from "next";
 import Link from "next/link";
-import { Container } from "react-bootstrap";
+import {Container, Image} from "react-bootstrap";
+import React from "react";
 
 const Home: NextPage = () => {
```

Comment:
**ease of understanding**: once again, try to make sure your formatting consistent - you use the `() => {}` form of function here, but use a different syntax to declare your `game()` function.